app/src/main/res/layout/simple_list_fragment.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.coordinatorlayout.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I don't think you use any of the CoordinatorLayout features here. So just keeping the RelativeLayout should be enough.

Reply:
I assumed that the FAB needs CoordinatatorLayout. If it doesn't, I can change it back.

Reply:
> I don't think you use any of the CoordinatorLayout features here. So just keeping the RelativeLayout should be enough.

I tried using reverting back to RelativeLayout. Turns out it we do need a CoordinatatorLayout for the SpeedDial to display properly.

Reply:
The old fragment uses a RelativeLayout: https://github.com/AntennaPod/AntennaPod/blob/daf6226483dccd168f25fee2d37ed2078e881504/app/src/main/res/layout/episodes_apply_action_fragment.xml

Reply:
> The old fragment uses a RelativeLayout: https://github.com/AntennaPod/AntennaPod/blob/daf6226483dccd168f25fee2d37ed2078e881504/app/src/main/res/layout/episodes_apply_action_fragment.xml

Ok, i'll try again.

Reply:
Hey @ByteHamster,
I've tried to use RelativeLayout but couldn't get the FAB to position correctly without using CoordinatorLayout. Unless you know how I think we should just stick with CoordinatorLayout.